sdcm/nemesis.py

```diff
@@ -1731,24 +1732,54 @@ def disrupt_nodetool_cleanup(self):
             for keyspace in test_keyspaces:
                 node.run_nodetool(sub_cmd="cleanup", args=keyspace)
 
-    def _prepare_test_table(self, ks='keyspace1', table=None):
-        ks_cfs = self.cluster.get_non_system_ks_cf_list(db_node=self.target_node)
-        table_exist = f'{ks}.{table}' in ks_cfs if table else True
+    def _prepare_test_table(self,
+                            ks: str = "keyspace1",
+                            table: str = "standard1",
+                            number_of_columns: int = 5,
+                            number_of_rows: int = 400000) -> None:
+        self.log.debug("Going to create a new keyspace `%s'...", ks)
+        self.tester.create_keyspace(
+            keyspace_name=ks,
+            replication_factor=self.tester.reliable_replication_factor,
+        )
 
-        test_keyspaces = self.cluster.get_test_keyspaces()
-        # if keyspace or table doesn't exist, create it by cassandra-stress
-        if ks not in test_keyspaces or not table_exist:
-            stress_cmd = "cassandra-stress write n=400000 cl=QUORUM -mode native cql3 " \
-                         f"-schema 'replication(factor={self.tester.reliable_replication_factor})' -log interval=5"
-            cs_thread = self.tester.run_stress_thread(
-                stress_cmd=stress_cmd, keyspace_name=ks, stop_test_on_failure=False, round_robin=True)
-            cs_thread.verify_results()
+        self.cluster.wait_for_schema_agreement()
+
+        self.log.debug("Going to create a new table `%s.%s'...", ks, table)
+        columns = {f'"C{index}"': "blob" for index in range(number_of_columns)}
+        self.tester.create_table(
+            name=table,
+            keyspace_name=ks,
+            key_type="blob",
+            read_repair=0.0,
+            compact_storage=True,
+            columns=columns,
+            scylla_encryption_options=self.tester.params.get("scylla_encryption_options"),
+            compaction=self.tester.params.get("compaction_strategy"),
+            sstable_size=self.tester.params.get("sstable_size"),
+        )
+
+        self.cluster.wait_for_schema_agreement()
+
+        self.log.debug("Populating table `%s.%s' with data...", ks, table)
+
+        with self.cluster.cql_connection_patient(self.target_node) as session:
+            session.default_consistency_level = ConsistencyLevel.QUORUM
+
+            query = f"INSERT INTO {ks}.{table} " \
+                    f"(key,{','.join(columns)}) " \
+                    f"VALUES ({('?,' * (number_of_columns + 1)).rstrip(',')});"
+            insert_query = session.prepare(query=query)
+
+            for _ in range(number_of_rows):
+                values = [uuid.uuid1().hex for _ in range(number_of_columns + 1)]
+                session.execute(query=insert_query, parameters=values)
```

Comment:
you might want to use `execute_concurrent_with_args`
https://github.com/scylladb/python-driver/blob/master/cassandra/concurrent.py#L231

should work a bit quicker.

Reply:
that would be great

Reply:
As agreed, the only thing we need in this workaround is to wait for the schema agreement in `disrupt_no_corrupt_repair`.
The proposed change is not needed.